### PR TITLE
Fix two log messsages

### PR DESCRIPTION
**Pull request by lbonn**
- the report in "Report not sent" does not exist anymore
- the putManifest disabled message sounds alarming, move it to DEBUG

### CODE REVIEW

File: src/libaktualizr/primary/sotauptaneclient.cc [hunk 1]
```diff
@@ -1189,7 +1188,7 @@ bool SotaUptaneClient::putManifestSimple() {
   // does not send event, so it can be used as a subset of other steps
   auto manifest = AssembleManifest();
   if (hasPendingUpdates()) {
-    LOG_INFO << "An update is pending, putManifest is disabled";
+    LOG_DEBUG << "An update is pending, putManifest is disabled";
```
Comment (doanac) [inline src/libaktualizr/primary/sotauptaneclient.cc]: thanks, that scared me for a while until i read the code :)

Comment (lbonn) [reply src/libaktualizr/primary/sotauptaneclient.cc]: haha, yes I've got that feedback a lot. Will try to think more about logging as an user interface :).

Comment (patrickvacek) [reply src/libaktualizr/primary/sotauptaneclient.cc]: This is fine, but is it worth expanding the message more, perhaps? "disabled" sounds off to me; perhaps "An update is pending. Skipping manifest upload until installation is complete." would be better?

Comment (lbonn) [reply src/libaktualizr/primary/sotauptaneclient.cc]: ok, reworked it again

**Pull request merged**